src/ManifoldsBase.jl

```diff
@@ -644,13 +627,39 @@ export allocate,
     embed!,
     geodesic,
     get_basis,
+    get_basis_default,
+    get_basis_diagonalizing,
+    get_basis_orthogonal,
+    get_basis_orthonormal,
+    get_basis_vee,
     get_component,
```

Comment:
I'm not sure if these functions should actually be exported, at least to reduce namespace pollution and confusion it may create.

Reply:
(this applies to pretty much all new exported functions)

Reply:
for get_component, I agree, for the others – I am not 100% sure.

Surely Manifolds should not export them - i.e. any high level stuff should not export the low level stuff, but the Interface itself? Hm, not sure. I mean for someone implementing an own manifold, they have to import them anyways, for anybody else it should be irrelevant, so yeah maybe we can leave them unescorted and document that properly in the motivation for the layers.

Reply:
We can always export them in a later version but removing an export is a breaking change, so I'd prefer to be conservative at this point.

Reply:
Sounds reasonable. Let‘s do that, keeps the number of exported functions clean - and users with own manifolds have to import them anyways (and all of these will still be properly documented).

Reply:
I removed exports of Layer 2 or 3 functions and will adopt the docs accordingly (where they still get a doc but that will be made more clear).